x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/e5_multilingual_callout/e5_multilingual_errors.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { HttpLogic } from '../../../../../../shared/http';
+
+import { SendEnterpriseSearchTelemetry } from '../../../../../../shared/telemetry';
+import { ML_NOTIFICATIONS_PATH } from '../../../../../routes';
+
+export function E5MultilingualErrors({ error }: { error: { title: string; message: string } }) {
+  const { http } = useValues(HttpLogic);
+
+  return (
+    <>
+      <SendEnterpriseSearchTelemetry action="error" metric="e5MultilingualModel-error" />
+      <EuiCallOut color="danger" iconType="error" title={error.title}>
+        <p>{error.message}</p>
+        <EuiLink href={http.basePath.prepend(ML_NOTIFICATIONS_PATH)} target="_blank">
+          {i18n.translate(
+            'xpack.enterpriseSearch.content.indices.pipelines.e5MultilingualCreateError.mlNotificationsLink',
+            {
+              defaultMessage: 'Machine Learning notifications',
```

Comment:
Is this the right default message on error?

Reply:
It is - in the error panel we instruct the user to go to the ML notification page where the error details can be seen. (The logic to show the error in the callout directly is not straightforward.)